data/headers/windows/c_payload_util/beacon.h

```diff
@@ -0,0 +1,69 @@
+/*
```

Comment:
This file came from https://github.com/Cobalt-Strike/bof_template where it's been published under the Apache 2.0 license.